src/pages/Zap/PoolCard.tsx

```diff
@@ -109,15 +115,26 @@ export const PoolCard: React.FC<Props> = ({
 
         {/* TODO show the connect wallet button */}
         {address && (
-          <ButtonPrimary
+          <ButtonEmpty
```

Comment:
What do you think about adding a border to the button? I think the manage button now looks quite different to the other buttons (so it's not clear that it is a button) - alternatively if you don't like the border, can we remove the padding on the sides so that the chevron is in line with the rest of the content on the right side and maybe make the text bold so it stands out more??

<img width="678" alt="Screenshot 2022-04-01 at 15 19 52" src="https://user-images.githubusercontent.com/20150449/161271622-382d2ace-9ac4-4e65-abee-66dd127a19a6.png">